Header header		# possible record keeping later

int32 cameraNum
---
string filename
